packages/aws-fargate/images/inspector/index.js

```diff
@@ -11,8 +11,6 @@ const app = new http.Server();
 const port = 3000;
 
 const { promises: fs } = require('fs');
-const fetch = require('node-fetch');
```

Comment:
These are used for the manual testing, for that we can use the global fetch, instead of node-fetch